### PR TITLE
More pages from wpgraphql.com /docs/

### PR DESCRIPTION
**Pull request by markkelnar**
What does this implement/fix? Explain your changes.
---------------------------------------------------
…

Does this close any currently open issues?
------------------------------------------
…

Any relevant logs, error output, GraphiQL screenshots, etc?
-------------------------------------
(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

Any other comments?
-------------------
…

Where has this been tested?
---------------------------
**Operating System:** …

**WordPress Version:** …

### CODE REVIEW

File: docs/debugging.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+
+```json
+  ...,
+  "tracing": {
+    "version": 1,
+    "startTime": "1600294443.335",
+    "endTime": "1600294443.381",
+    "duration": 45998,
+    "execution": {
+      ...
```
Comment (jasonbahl) [inline docs/debugging.md:171]: Github is syntax highlighting this like an error. 

Want to see if there actually is some issue or if Github is just being a turd.

Comment (jasonbahl) [reply docs/debugging.md:171]: Screen Shot 2021-05-04 at 2 26 10 PM

**Review by jasonbahl — APPROVED** [review]
The `debugging.md` file had some syntax highlighting issues, according to Github. 

Just want to check if that's actually an issue, or if Github is just being a turd. 

On initial glance, this looks 🔥 

Reminding me how much content I've written 😄 , but also how much STILL needs to be written 🙈

**Comment by markkelnar** [comment]
> tial glance, this looks

It's the extra ... in the json code block causing the githib diff to go red.

**Pull request merged**